### ISSUE TITLE
bad pip url for BiblioPixelAnimations

### ISSUE THREAD

**Issue by praeluceo** [ManiacalLabs/PixelWeb#1]
When launching run-pixelweb it runs a pip install on BiblioPixel's packages. The bibliopixelanimations url is incorrect.

$ sudo run-pixelweb
Installing BiblioPixelAnimations

and then it fails, attempting to manually install BiblioPixelAnimations gives more feedback:

```
# pip install --no-deps BiblioPixelAnimations -v
Collecting BiblioPixelAnimations
  Getting page https://pypi.python.org/simple/bibliopixelanimations/
  Starting new HTTPS connection (1): pypi.python.org
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropri
ately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  "GET /simple/bibliopixelanimations/ HTTP/1.1" 404 28063
  Could not fetch URL https://pypi.python.org/simple/bibliopixelanimations/: 404 Client Error: Not Found - skipping
  1 location(s) to search for versions of BiblioPixelAnimations:
  * https://pypi.python.org/simple/bibliopixelanimations/
  Getting page https://pypi.python.org/simple/bibliopixelanimations/
  "GET /simple/bibliopixelanimations/ HTTP/1.1" 404 28063
  Could not fetch URL https://pypi.python.org/simple/bibliopixelanimations/: 404 Client Error: Not Found - skipping
  Could not find a version that satisfies the requirement BiblioPixelAnimations (from versions: )
Cleaning up...
No matching distribution found for BiblioPixelAnimations
Exception information:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pip/basecommand.py", line 211, in main
    status = self.run(options, args)
  File "/usr/local/lib/python2.7/dist-packages/pip/commands/install.py", line 294, in run
    requirement_set.prepare_files(finder)
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_set.py", line 334, in prepare_files
    functools.partial(self._prepare_file, finder))
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_set.py", line 321, in _walk_req_to_install
    more_reqs = handler(req_to_install)
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_set.py", line 461, in _prepare_file
    req_to_install.populate_link(finder, self.upgrade)
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_install.py", line 250, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/local/lib/python2.7/dist-packages/pip/index.py", line 571, in find_requirement
    'No matching distribution found for %s' % req
DistributionNotFound: No matching distribution found for BiblioPixelAnimations
```

The path does not exist on https://pypi.python.org/simple, the correct pip location would be:
# pip install  https://github.com/ManiacalLabs/BiblioPixelAnimations/archive/master.zip --upgrade

BiblioPixelAnimations should either be added under https://pypi.python.org/simple/bibliopixel/ or given its own entry, or the script should be updated to point to the correct url.

**Comment by adammhaile** [comment]
What is the actual error it gives when you call run-pixelweb and it tries to install BiblioPixelAnimations? The path in the code is correct and working on my end. The error you did post is expected because BPA is not on PyPi and will not be for a variety of reasons. Mainly, it's an ever changing repository and the intent is to always just install it directly from github.

**Comment by praeluceo** [comment]
I have to work around a proxy at work, so I manually specify --proxy= to pip to get things installed. I invoked run-pixelweb and when it would state that it was fetching a package, I would cancel run-pixelweb and execute a "pip --proxy install" with that package name. When I tried doing a pip install BiblioPixelAnimations however it failed because https://pypi.python.org/simple/bibliopixelanimations/ doesn't exist. But now that I investigated a little, I see that line 28 in bootstrap.py correctly calls the long url for animations, so this is not a bug, it's just my failure to read through the code when pip install failed. If I'd looked, I could have found the correct path. Sorry about that!
